### ISSUE TITLE
Searching with SearchBox removes default query

### ISSUE THREAD

**Issue by probabble** [searchkit/searchkit#526]
A default query created like this:
```
searchkit.addDefaultQuery(query => {
  return query.addQuery(BoolMust(TermQuery("live_filter", true)));
});
```
Works fine until a user begins typing into the SearchBox, at which point the default query is removed from the request to ElasticSearch.

**Comment by ssetem** [comment]
hi @probabble, just tested locally and also see the same issue.

Will try to fix thanks!

**Comment by ssetem** [comment]
@probabble the fix is in beta version 2.3.0-0

possible to test when you get a chance?

thanks!
